File: python/8 kyu/120.py
# https://www.codewars.com/kata/5a0be7ea8ba914fc9c00006b/train/python

# Passed

def sakura_fall(v):
    if v <= 0:
        return 0

    time_in_seconds = (5 * 80) / v
    return time_in_seconds

output = sakura_fall(10)
print(output)

File: nodejs/7 kyu/120.js
// https://www.codewars.com/kata/589478160c0f8a40870000bc/train/javascript

// Paased

function arrowArea(a, b) {
  let rectangleArea = a * b;
  let arrowArea = rectangleArea / 4;
  return arrowArea;
}

const output = arrowArea(4, 2);
console.log(output);